base_force_record_noupdate/tests/test_base_force_record_noupdate.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2024 Camptocamp (https://www.camptocamp.com).
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+
+from odoo.tests import TransactionCase
+
+MODULE = "base_force_record_noupdate"
+
+
+class TestBaseForceRecordNoupdate(TransactionCase):
+    def test_00_test_model_create_noupdate_propagation(self):
+        # NB: we're only testing models created through UI, the feature is not
```

Comment:
I don't get this comment. Create is overridden, what's the probl?

Reply:
What I mean is that if we create a new model via Python code, we still need to use the UI to set this field to `True`.
The `create` override only applies for models created through the UI: if someone plans to create an all-new model from the Py code, it'll trigger an `ir.model.create()`, but the newly created record will have `force_noupdate = False` (at least until the record is updated via a `write()`).
Thus, the only way to test the `ir.model.create()` override is to mimic the manual creation of a new model.